cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/spark/SparkProgramRunner.java

```diff
@@ -56,12 +58,17 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.Closeable;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
 
 /**
  * Runs {@link Spark} programs
  */
-public class SparkProgramRunner implements ProgramRunner {
+public class SparkProgramRunner extends AbstractProgramRunnerWithPlugin {
```

Comment:
I'll leave it to happen organically as a TODO most likely ended up describing something no longer accurate in future.